examples/shell/shell_common/common.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <Common.h>
+
+chip::Messaging::ExchangeManager gExchangeMgr;
```

Comment:
please avoid names of common/util/support/helpers.

This could be 'global_state.cpp' or 'globalObjects'.

use Manager instead of Mgr to optimize for readabiltiy instead fo typing speed.

Reply:
Acked